### PR TITLE
Add local dependency install script and documentation

### PR DESCRIPTION
**Pull request by mapingo**
How to build and deploy Microservice Framework locally without access to build repository.  Includes useful script for building and deploying dependencies into the local m2 cache.

### CODE REVIEW

File: install-dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+cd ../file-service
+installVersions ${file_service_versions[@]}
+
+cd ../json-schema
+installVersions ${json_schema_versions[@]}
+
+cd ../wildfly-maven-plugin
+installVersions ${wildfly_maven_plugin_versions[@]}
+
+cd ../microservice-framework
```
Comment (purple52) [inline install-dependencies.sh]: Should this last one be an underscore instead of a dash?

Comment (mapingo) [inline install-dependencies.sh]: Yes, I missed that, corrected.

**Comment by mapingo** [comment]
I am not sure what Coveralls is on about here?

**Comment by purple52** [comment]
Yeah, no idea why Coveralls is upset. Let's see what happens when its merged. Thanks!

**Pull request merged**